src/main/scala/io/dronekit/cloud/AWSWrapper.scala

```diff
@@ -170,7 +170,7 @@ class AWSWrapper(S3Client: AmazonS3Client = S3.client)
   }
 
   def streamUpload(s3url: S3URL): Sink[ByteString, Future[CompleteMultipartUploadResult]] = {
-    multipartUploadTransform(s3url).to(Sink.head)
+    multipartUploadTransform(s3url).fold(Seq.empty[UploadPartResult])(_ :+ _).to(Sink.head)
```

Comment:
Since `.to` returns only the materialized value of the first Flow, why not `multipartUploadTransform(s3url).to(Sink.ignore)` instead of making a `Seq` to throw away?